melodic-manatees/early_internet/background_app/background_utility.py

```diff
@@ -0,0 +1,36 @@
+from PIL import Image
+import os.path
+from django.core.exceptions import ValidationError
+
+
+def resolution_checker(image):
+    '''check resolution of uploaded images, reject if too small or too big'''
+    im = Image.open(image)
```

Comment:
Try to avoid variable names like `im`. I'm guesing you're using `im` here because `image` is already taken. It'd be better to rename the function argument to `image_path` and let this variable be named `image`, but even if that wasn't an option, try to find a full variable name instead of these abbreviated ones, it really helps readability.

One way to avoid variable name collision is to suffix an underscore, like `image_`